dpsprep/outline.py

```diff
@@ -1,23 +1,33 @@
 from loguru import logger
 from pdfrw import PdfName, PdfDict, IndirectPdfDict
 import djvu.sexpr
+import re
 
 from .sexpr import SExpressionVisitor
 
 
 # Based on
 # https://github.com/pmaupin/pdfrw/issues/52#issuecomment-271190546
 class OutlineTransformVisitor(SExpressionVisitor):
+    def __init__(self, toc_pg_offset : int, total_pages : int):
```

Comment:
This code fails the type checker - try running `make lint`. 

1. These two type hints should also be added to the class itself (dataclasses can then be used to avoid writing a trivial constructor, but given that the other code here doesn't use dataclasses, I wouldn't go as far as to recommend it).

2. Please use consistent styling. The other code has no space before the colons.

Reply:
Passed `make lint` on my side. Fixed the inconsistent styling.